templates/webserver/webserver-deployment.yaml

```diff
@@ -88,22 +88,22 @@ spec:
           ports:
             - name: webserver-ui
               containerPort: {{ .Values.ports.airflowUI }}
-          livenessProbe:
-            httpGet:
-              path: /health
-              port: {{ .Values.ports.airflowUI }}
-            initialDelaySeconds: {{ .Values.webserver.livenessProbe.initialDelaySeconds | default 15 }}
-            timeoutSeconds: {{ .Values.webserver.livenessProbe.timeoutSeconds | default 30 }}
-            failureThreshold: {{ .Values.webserver.livenessProbe.failureThreshold | default 20 }}
-            periodSeconds: {{ .Values.webserver.livenessProbe.periodSeconds | default 5 }}
-          readinessProbe:
-            httpGet:
-              path: /health
-              port: {{ .Values.ports.airflowUI }}
-            initialDelaySeconds: {{ .Values.webserver.readinessProbe.initialDelaySeconds | default 15 }}
-            timeoutSeconds: {{ .Values.webserver.readinessProbe.timeoutSeconds | default 30 }}
-            failureThreshold: {{ .Values.webserver.readinessProbe.failureThreshold | default 20 }}
-            periodSeconds: {{ .Values.webserver.readinessProbe.periodSeconds | default 5 }}
+#          livenessProbe:
+#            httpGet:
+#              path: /health
```

Comment:
These should still be enabled

Reply:
Agreed, I think our `base_url` needs to be accounted for here.